qa/sc_evm_native_forger_v2.py

```diff
@@ -595,6 +608,22 @@ def run_test(self):
             logging.info("Expected exception thrown: {}".format(err))
             assert_true("Forger doesn't exist" in str(err))
 
+        # Check stakeStart
+        method_stake_start = "stakeStart(bytes32,bytes32,bytes1,address)"
+        data_input = forger_v2_native_contract.raw_encode_call(method_stake_start,
+                                                               forger_1_sign_key_to_bytes,
+                                                               forger_1_vrf_pub_key_to_bytes[0:32],
+                                                               forger_1_vrf_pub_key_to_bytes[32:],
+                                                               "0x" + evm_address_sc_node_1)
+        result = sc_node_1.rpc_eth_call(
+            {
+                "to": "0x" + FORGER_STAKE_V2_SMART_CONTRACT_ADDRESS,
+                "from": add_0x_prefix(evm_address_sc_node_1),
+                "input": data_input
+            }, "latest"
+        )
+        assert_equal(decode(['uint32'], hex_str_to_bytes(result['result'][2:]))[0], VERSION_1_4_FORK_EPOCH)
```

Comment:
It works here, but actually it should be decoded as an an `int32`. Moreover maybe we could add a test using an address which did not delegate anything and checking that it returns `-1`.

Reply:
Thanks, fixed and added a check